### PR TITLE
[Fix #72] Hero screen size for all viewports

### LINKED ISSUE #72: Hero Screen is much larger than the browser height
I think the Hero section should have a maximum height of 100vh whereas it is exceeding it.

### PR DESCRIPTION
**Pull request by sagarchoudhary96**
screenshot from 2017-08-03 12-11-24

The hero screen was overflowing on mobile screens so i made it's height to 100vh so that it looks same on all devices. Below is the updated screenshot:

screenshot from 2017-08-03 12-15-13

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@sagarchoudhary96 resolve the conflicts. And after that give a **live link** of the changes made by you for reviewers to see the changes.

**Comment by sagarchoudhary96** [comment]
you can view the updated version [here.](https://sagarchoudhary96.github.io/inpycon2017/index.html)

**Comment by sagarchoudhary96** [comment]
@theskumar fixed the conflicts and provided live version link too

**Comment by sagarchoudhary96** [comment]
@realslimshanky @theskumar please see to it

**Comment by realslimshanky** [comment]
@sagarchoudhary96 problem with your fix is that it'll now work with the screen with small devices.
All the buttons get cramped into little space on the screen and it does not look good.
Exhibit A
image

**Comment by sagarchoudhary96** [comment]
@realslimshanky but i think it looks better and the button were having that much magin before too i just made the  blue scren size too 100vh which didn't changed any no decreased the distances between buttons too

**Comment by realslimshanky** [comment]
Sure. I would wait for others to review.

**Comment by sagarchoudhary96** [comment]
@realslimshanky is anybody looking to these PR's ?

**Comment by realslimshanky** [comment]
@CuriousLearner @sayanchowdhury would you like to have a look into this?

**Comment by sagarchoudhary96** [comment]
@realslimshanky seems like no one looked at it. It would be great if you can see to it and let me know if required what changes to be made ?

File: css/main.css [hunk 1]
```diff
@@ -1192,41 +1192,40 @@ li.tg-rss a{background:#ff6600;}
...
-    rgba(68,67,73,0.85) 0%,
-    rgba(70,66,73,0.85) 1%,
-    rgba(217,8,69,0.85) 60%);
-  background:
-    linear-gradient(to bottom,
-    rgba(68,67,73,0.85) 0%,
-    rgba(70,66,73,0.85) 1%,
-    rgba(217,8,69,0.85) 60%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#d9444349', endColorstr='#d9d90845',GradientType=0 );
+	width:100%;
```
Comment (CuriousLearner) [inline css/main.css]: Please use 2 spaces indentation for the files.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Left review inline.

Please ensure all the code you add is consistent with the code that is already there.

**Comment by sagarchoudhary96** [comment]
@CuriousLearner I have made the required changes.

**Comment by sagarchoudhary96** [comment]
@realslimshanky @CuriousLearner would be great if you can look to it

File: css/main.css [hunk 2]
```diff
@@ -4941,3 +4940,6 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
   color: white;
 }
 
+.height-100vh {
```
Comment (SaptakS) [inline css/main.css]: What is the use of this class? Can you please explain? I don't see it being used anywhere. If not needed please remove.

**Review by SaptakS — CHANGES_REQUESTED** [review]

**Comment by sagarchoudhary96** [comment]
@SaptakS removed it. I created it to use it in the index.html but later on i fixed it by making changes in css only. Just forgot to remove it. now it is removed.

File: css/responsive.css [hunk 3]
```diff
@@ -1038,4 +1033,4 @@
 	}
 	.tg-headervfour .tg-navigationarea .tg-btnbookseat{float:left;}
 }
-@media (max-width:320px){}
\ No newline at end of file
+@media (max-width:320px){}
```
Comment (realslimshanky) [inline css/responsive.css]: What's the need of this media query?

Comment (sagarchoudhary96) [reply css/responsive.css]: this was not added by me it was already there in the code.

Comment (realslimshanky) [reply css/responsive.css]: Please remove this line.

Comment (sagarchoudhary96) [reply css/responsive.css]: @realslimshanky removed it

**Comment by sagarchoudhary96** [comment]
@realslimshanky done👍

**Comment by sagarchoudhary96** [comment]
@CuriousLearner it has been too long please see to it

**Pull request closed without merging**